### DISCUSSION TITLE
Check Template Overrides Error

### DISCUSSION THREAD

**Discussion by rkonikpl** [akeeba/panopticon#568] [Q&A (Community Support)]
I'm not good at programming so I prefer to write it here.
I have a problem with one website that has information about overwrites. When I click "Check Template Overrides" I get an error
Here is its content. Of course I anonymized some paths

`TypeError: Awf\Html\Helper\Basic::date(): Argument #1 ($input) must be of type string, null given, called in _PATH_/public_html/tmp/compiled_templates/592a9097491011e64bd521c304c7c38bd6d2be98.php on line 108 and defined in _PATH_/public_html/vendor/akeeba/awf/src/Html/Helper/Basic.php:92 Stack trace: #0 _PATH_/public_html/tmp/compiled_templates/592a9097491011e64bd521c304c7c38bd6d2be98.php(108): Awf\Html\Helper\Basic->date(NULL, 'Y-m-d H:i') #1 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/View.php(1301): include('/home/klient.dh...') #2 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/View.php(1231): Awf\Mvc\View->includeTemplateFile(Array) #3 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/View.php(745): Awf\Mvc\View->processTemplate(Array) #4 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/View.php(638): Awf\Mvc\View->loadAnyTemplate('Overrides/defau...') #5 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/View.php(562): Awf\Mvc\View->loadTemplate(NULL) #6 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/Controller.php(447): Awf\Mvc\View->display() #7 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/DataController.php(73): Awf\Mvc\Controller->display() #8 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/Controller.php(375): Awf\Mvc\DataController->browse() #9 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/DataController.php(57): Awf\Mvc\Controller->execute('browse') #10 _PATH_/public_html/src/Controller/Overrides.php(72): Awf\Mvc\DataController->execute('browse') #11 _PATH_/public_html/vendor/akeeba/awf/src/Dispatcher/Dispatcher.php(125): Akeeba\Panopticon\Controller\Overrides->execute('default') #12 _PATH_/public_html/src/Dispatcher.php(36): Awf\Dispatcher\Dispatcher->dispatch() #13 _PATH_/public_html/vendor/akeeba/awf/src/Application/Application.php(236): Akeeba\Panopticon\Dispatcher->dispatch() #14 _PATH_/public_html/src/Application.php(419): Awf\Application\Application->dispatch() #15 _PATH_/public_html/index.php(35): Akeeba\Panopticon\Application->dispatch() #16 _PATH_/public_html/index.php(38): {closure}() #17 {main} Next ErrorException: Awf\Html\Helper\Basic::date(): Argument #1 ($input) must be of type string, null given, called in _PATH_/public_html/tmp/compiled_templates/592a9097491011e64bd521c304c7c38bd6d2be98.php on line 108 [View template: ROOT/tmp/compiled_templates/592a9097491011e64bd521c304c7c38bd6d2be98.php line 92, compiled from ROOT/ViewTemplates/Overrides/default.blade.php] in _PATH_/public_html/vendor/akeeba/awf/src/Html/Helper/Basic.php:92 Stack trace: #0 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/View.php(1235): Awf\Mvc\View->handleViewException(Object(TypeError), 1) #1 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/View.php(745): Awf\Mvc\View->processTemplate(Array) #2 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/View.php(638): Awf\Mvc\View->loadAnyTemplate('Overrides/defau...') #3 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/View.php(562): Awf\Mvc\View->loadTemplate(NULL) #4 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/Controller.php(447): Awf\Mvc\View->display() #5 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/DataController.php(73): Awf\Mvc\Controller->display() #6 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/Controller.php(375): Awf\Mvc\DataController->browse() #7 _PATH_/public_html/vendor/akeeba/awf/src/Mvc/DataController.php(57): Awf\Mvc\Controller->execute('browse') #8 _PATH_/public_html/src/Controller/Overrides.php(72): Awf\Mvc\DataController->execute('browse') #9 _PATH_/public_html/vendor/akeeba/awf/src/Dispatcher/Dispatcher.php(125): Akeeba\Panopticon\Controller\Overrides->execute('default') #10 _PATH_/public_html/src/Dispatcher.php(36): Awf\Dispatcher\Dispatcher->dispatch() #11 _PATH_/public_html/vendor/akeeba/awf/src/Application/Application.php(236): Akeeba\Panopticon\Dispatcher->dispatch() #12 _PATH_/public_html/src/Application.php(419): Awf\Application\Application->dispatch() #13 _PATH_/public_html/index.php(35): Akeeba\Panopticon\Application->dispatch() #14 _PATH_/public_html/index.php(38): {closure}() #15 {main}`

**Comment by rkonikpl** [comment]
I discovered that the problem occurs when the website has the default "Helix Ultimate" template and a template from joomshaper. My templates do not have this error.

**Reply by nikosdion** [reply]
You need to open the referenced file, `tmp/compiled_templates/592a9097491011e64bd521c304c7c38bd6d2be98.php`, in a text editor and tell me which file it says it's compiled from. This information is in the first few lines of the file.

**Reply by rkonikpl** [reply]
first line. That was it?
`public_html/ViewTemplates/Overrides/default.blade.php`

**Reply by nikosdion** [reply]
I believe that's fixed with https://github.com/akeeba/panopticon/commit/dd8ac9eaf1db6f3b3015434abd31119bc47def95
